core-js/src/main/javascript/cdf/components/filter/views/Abstract.js

```diff
@@ -219,10 +225,14 @@ define([
     },
     setScrollBarAt: function ($tgt) {
       if (this._scrollBar != null) {
-        this._scrollBar.scrollIntoView($tgt);
+        this._scrollToPosition($tgt);
       }
       return this;
     },
+    /**
+     * function to be overridden by addScrollBar depending on the scrollbar engine 
+     */
+    _scrollToPosition: function (Position){},
```

Comment:
Personally I'm not very fond of dynamically overriding a method.
Perhaps you could write a proper ```this. _scrollToPosition``` that checks which scrollbar implementation we should be using.

This solution works, but ideally, this should be abstracted away, by delegating to a class that deals with the scrollbar, and whose constructor is chosen  dynamically.